synapse/lib/ast.py

```diff
@@ -1450,17 +1450,18 @@ async def yieldFromValu(self, runt, valu):
             return
 
         # a little DWIM on what we get back...
-        # ( most common case will be stormtypes libs agenr -> iden|buid|nid )
+        # ( most common case will be stormtypes libs agenr -> iden|buid )
         # buid list -> nodes
         if isinstance(valu, bytes):
-            valulen = len(valu)
-            if valulen == 8:
-                if (node := await runt.view.getNodeByNid(valu)) is not None:
-                    yield node
+            if (node := await runt.view.getNodeByBuid(valu)) is not None:
+                yield node
 
-            elif valulen == 32:
-                if (node := await runt.view.getNodeByBuid(valu)) is not None:
-                    yield node
+            return
+
+        # nid -> node
+        if isinstance(valu, int):
```

Comment:
Should we make this the first case we check for since it'll be most common and help a hair with perf?